config/kafka-instance-types-configuration.yaml

```diff
@@ -32,7 +33,8 @@ supported_instance_types:
       quotaConsumed: 1
       quotaType: "rhosak"
       capacityConsumed: 1
-  - id: eval
+  - id: developer
+    display_name: Trial
```

Comment:
Why putting the 'display name' into the configuration? IMO the display name is decided by the consumer, here we should have only the `id`

Reply:
ignore this.